Trell/Protos/TrellServer.proto

```diff
@@ -31,9 +31,6 @@ service TrellServer {
 
   // -------------------- START INTERNAL API -----------------------------
 
-  // Called from the worker (on a user's behalf) to log.
-  rpc Log(LogRequest) returns (google.protobuf.Empty);
```

Comment:
If it does make sense to remove `Log`, it will make sense to remove LogRequest and any other messages/types it depends on.